src/App.jsx

```diff
@@ -1,23 +1,23 @@
-import { Route, Routes } from 'react-router-dom';
-import LoginHomepage from './pages/LoginHomepage/LoginHomepage';
-import ArticlesMain from './pages/ArticlesMain/ArticlesMain';
-import IndividualArticle from './pages/IndividualArticle/IndividualArticle';
-import Header from './components/Header/Header';
-import './App.css';
+import { Route, Routes } from "react-router-dom";
+import LoginHomepage from "./pages/LoginHomepage/LoginHomepage";
+import ArticlesMain from "./pages/ArticlesMain/ArticlesMain";
+import IndividualArticle from "./pages/IndividualArticle/IndividualArticle";
+import Header from "./components/Header/Header";
+import "./App.css";
 
 export default function App() {
   return (
-    <div className='app-container'>
-      <div className='app-container__header-container'>
+    <div className="app-container">
+      <div className="app-container__header-container">
         <Header />
       </div>
-      <div className='app-container__routes-container'>
+      <div className="app-container__routes-container">
         <Routes>
-          <Route path='/' element={<LoginHomepage />}></Route>
-          <Route path='/articles' element={<ArticlesMain />}></Route>
-          <Route path='/article/:article_id' element={<IndividualArticle />} />
+          <Route path="/" element={<LoginHomepage />} />
+          <Route path="/articles" element={<ArticlesMain />} />
+          <Route path="/article/:article_id" element={<IndividualArticle />} />
           <Route
-            path='/article/:article_id/comments'
+            path="/article/:article_id/comments"
             element={<IndividualArticle />}
```

Comment:
With things like this where all the quotes seem to change, often teams will use a prettier config file `.prettierrc` to create specific rules for formatting so that one person doesn't change all the quotes to single quotes whilst another changes them all to double quotes. May be worth having a look at in future if you find this occurring more :) 

https://prettier.io/docs/en/configuration.html